### PR TITLE
B001. Add database model and migrations

### PR DESCRIPTION
**Pull request by ManuelLoaizaV**
Tables:
- [x] User
- [x] Role
- [x] Dish
- [x] Ingredient
- [x] DishIngredient
- [x] Order
- [x] OrderStatus
- [x] OrderDish

Chores:
- [x] Create diagram in drawio.
- [x] Make migrations.

### CODE REVIEW

**Review by AngelCrispin — APPROVED** [review]
Ta gud

File: src/api/models.py [hunk 1]
```diff
@@ -72,3 +72,14 @@ class Order(db.Model):
...
+
+class Dish(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    name = db.Column(db.String(255), unique=True, nullable=False)
+    description = db.Column(db.String(255), nullable=False)
+    image_url = db.Column(db.String(255))
+    price = db.Column(db.Float, nullable=False)
+    discount_percentage = db.Column(db.Integer, nullable=False)
+    cooking_time = db.Column(db.Integer)
+    created_at = db.Column(db.DateTime, nullable=False, default=datetime.now)
```
Comment (AngelCrispin) [inline src/api/models.py]: Falta quantity

**Review by AngelCrispin — CHANGES_REQUESTED** [review]

File: src/api/models.py (hunk 1 above)
Comment (ManuelLoaizaV) [reply src/api/models.py]: Nice catch, thanks!

**Comment by ROZZYNUEZ14** [comment]
👍

**Review by ROZZYNUEZ14 — APPROVED** [review]

**Review by AngelCrispin — APPROVED** [review]
LGFA

**Pull request merged**